Makefile

```diff
@@ -46,6 +46,11 @@ help:
 	@echo '  # how to build latest checked-out sources and run an example:'
 	@echo '  $$ make ci'
 	@echo
+	@echo '  # how to run a specific example:'
+	@echo '  $$ make run-example EXAMPLE=reana-demo-helloworld'
+	@echo '  # how to run all REANA examples:'
+	@echo '  $$ make run-example'
```

Comment:
`run-example` would be the only two-word target amongst the rest... That's why I used `example` as a single word.

(It may not be pretty, but `test` was already taken for the unit test... unless we think of renaming the two...)